packages/extension-commerce-bm-seo/src/setup-app.ts

```diff
@@ -92,22 +92,32 @@ class CommerceBmSeo extends ApplicationExtension<Config> {
         // Make SEO GET Url Mapping API call
         const config = this.getConfig()
         const shopperSeo = await getShopperSeoClient(locals, config)
-        const urlMapping = await shopperSeo.getUrlMapping({
+        let urlMapping = await shopperSeo.getUrlMapping({
             parameters: {urlSegment: locals.originalUrl}
         })
         console.log('--- url mapping', urlMapping)
 
+        // DEBUG
+        urlMapping = {resourceType: 'category'}
+
         if (!urlMapping.resourceType) {
             return Promise.resolve([])
         }
 
         // Transform URL mapping to serialized route
         const requestURL = new URL(locals.originalUrl, getAppOrigin(locals))
         const componentName =
-            config.resourceTypeToComponentMap[urlMapping.resourceType as keyof typeof config.resourceTypeToComponentMap]
+            config.resourceTypeToComponentMap[
+                urlMapping.resourceType as keyof typeof config.resourceTypeToComponentMap
+            ]
+
         return Promise.resolve([
             {
                 path: requestURL.pathname,
+                // Placeholder component that will be replaced later during beforeRouteMatch
+                component: Sample,
+                // Still need this `componentName` for later (during beforeRouteMatch).
+                // `displayName` will be a long name, after component has been wrapped multiple times.
                 componentName,
```

Comment:
Do we still want to introduce a new field componentName to RouteProps? I guess it's a shortcut to access the name of the base component without having to parse the displayName that's wrapped like `WithErrorHandling(withRouter(routeComponent(ProductList)))`